show/main.py

```diff
@@ -1046,6 +1047,15 @@ def logging(process, lines, follow, verbose):
         run_command(cmd, display_cmd=verbose)
 
 
+def get_uptime():
+    sys_uptime_cmd = "uptime"
+    sys_uptime = subprocess.Popen(sys_uptime_cmd, shell=True, text=True, stdout=subprocess.PIPE)
+    # The 'uptime' command displays the current time as the 1st entry "hh:mm:ss {more info}"
```

Comment:
please change to 
Linux 'uptime' command displays the current time as the 1st entry "hh:mm:ss <more info>"
Split the output space character to remove the time from the output.